From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sunu Aziz Rahayu <sunuazizrahayu@users.noreply.github.com>
Date: Fri, 26 Sep 2026 00:00:00 +0700
Subject: [PATCH] cmds: drop vibrator-lge depending on removed LGE HAL

vendor.lge.hardware.vibrator@1.0 sudah dihapus (ikut
android.hardware.vibrator@1.0 yang tidak ada lagi di Lineage 23.2),
sehingga Soong gagal:

  "vibrator-lge" depends on undefined module
  "vendor.lge.hardware.vibrator@1.0"

vibrator-lge hanya binary debug untuk perangkat LG lama, tidak
direferensikan base.mk maupun rw-system.sh, jadi hapus saja bloknya.

---
 cmds/Android.bp | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/cmds/Android.bp b/cmds/Android.bp
index 0000000..0000000 100644
--- a/cmds/Android.bp
+++ b/cmds/Android.bp
@@ -36,19 +36,7 @@ cc_binary {
 	],
 	host_supported: true,
 }

-cc_binary {
-	name: "vibrator-lge",
-	srcs: [
-		"vibrator-lge.cpp",
-	],
-	shared_libs: [
-		"vendor.lge.hardware.vibrator@1.0",
-		"libutils",
-		"libhidlbase",
-	],
-}
-
 cc_binary {
 	name: "mtk-sms-fwk-ready",
 	srcs: [
--
2.54.0
